src/uwtools/tests/drivers/test_schism.py

```diff
@@ -36,14 +37,19 @@ def cycle():
 
 @fixture
 def driverobj(config, cycle):
-    return schism.SCHISM(config=config, cycle=cycle, batch=True)
+    return SCHISM(config=config, cycle=cycle, batch=True)
 
 
 # Tests
 
 
-def test_SCHISM(driverobj):
-    assert isinstance(driverobj, schism.SCHISM)
+def test_SCHISM():
+    for method in [
+        "_driver_config",
+        "_taskname",
+        "_validate",
+    ]:
+        assert getattr(SCHISM, method) is getattr(Assets, method)
```

Comment:
Note that the SCHISM and WW3 driver derive from the `Assets` base class and so have a shorter list of inherited methods.